native/database.cc

```diff
@@ -20,13 +20,14 @@
 #include "thread_utils.hh"
 
 constexpr int QUEUE_SIZE = 1000;
-constexpr absl::CivilDay epoch(1900);
+constexpr absl::CivilDay epoch(1800);
+constexpr absl::CivilDay legacy_epoch(1900);
 
 constexpr int seconds_per_minute = 60;
 constexpr int minutes_per_hour = 60;
 constexpr int hours_per_day = 24;
 
-constexpr uint32_t current_version = 2;
+constexpr uint32_t current_version = 3;
```

Comment:
Will this branch support prior versions? for example, can I keep using the v7 extract after merging this branch into main?